RFS/src/main/java/com/rfs/cms/ScopedWorkCoordinatorHelper.java

```diff
@@ -0,0 +1,43 @@
+package com.rfs.cms;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class ScopedWorkCoordinatorHelper {
+
+    public final IWorkCoordinator workCoordinator;
+    final ProcessManager processManager;
+
+    public ScopedWorkCoordinatorHelper(IWorkCoordinator workCoordinator, ProcessManager processManager) {
+        this.workCoordinator = workCoordinator;
+        this.processManager = processManager;
+    }
+
+    public interface WorkItemGetter {
+        @NonNull IWorkCoordinator.WorkAcquisitionOutcome apply(IWorkCoordinator wc);
```

Comment:
```suggestion
        @NonNull IWorkCoordinator.WorkAcquisitionOutcome get(IWorkCoordinator wc);
```

Reply:
Or maybe `tryAcquire(...)` is a better fit?

Reply:
Good point.  Changed.